src/http/services/execute.service.ts

```diff
@@ -47,10 +47,12 @@ export class ExecuteService {
         switch (execution.command) {
           case ExecuteCommandTypes.Brightness:
           case ExecuteCommandTypes.OnOff:
+		  case ExecuteCommandTypes.LockUnlock:
```

Comment:
`LockUnlock` command should be handled differently. It comes with a `lock` param that should be mapped to `isLocked` state property.
https://developers.google.com/assistant/smarthome/traits/lockunlock#device-commands